polyply/data/martini3/PPE.martini3.ff

```diff
@@ -0,0 +1,52 @@
+; General remarks:
+; After "polyplying" PPE to the desired polymer length, some manual adjustments are recommended:
+; 1) PPE ends with an aromatic ring and not with a triple bond: Delete last bead of topology and coordinate file and all respective potentials using that bead. 
+; 2) To prevent bending over of last and first aromatic ring: Add an additional bond angle potential at both polymer ends. (k=50,phi0=165)
+;    (e.g. bond angle potential between 1-4-8 and between End-(End-4)-(End-8) ).
+; 3) For furher explanation, please look at https://doi.org/10.1039/D1CP04237H.
+
+[ citations ]
+Martini3
+polyply
+PPEs
+
+[ moleculetype ]
+PPE    3
+
+[ atoms ]
+;  nr    type       resnr  residu    atom    cgnr        charge   
+   1     TC5           1  PPE        BB1       1     0.0   
+   2     TC5           1  PPE        BB2       2     0.0
+   3     TC5           1  PPE        BB3       3     0.0
+   4     TC4           1  PPE        BB4       4     0.0  
+     
+[ bonds ]
+;  ai  aj   funct
+   3    4   1   0.234 9000 		{"comment": "bond aromatic ring-triple bond"}
+
+[ constraints ]
+;  ai  aj   funct
+    1   2   1    0.325   		{"comment": "constr. aromatic ring"}
+    1   3   1    0.325   		{"comment": "constr. aromatic ring"}
+    2   3   1    0.325   		{"comment": "constr. aromatic ring"}
+    
+[ angles ]
+   1    3    4     1    143    550  	{"comment": "angle aromatic ring-triple bond"}
+   2    3    4     1     83    650   	{"comment": "angle aromatic ring-triple bond"}
+   
+[ dihedrals ]
+    4    2    1   3    2  0    50   	{"comment": "improper dihedral left turn"}
+    
+[ link ]
+resname "PPE|PPEter"
```

Comment:
This link only concerns PPE and not PPEter or am I mistaken? If so then you can remove the second residue name